config/initializers/sentry.rb

```diff
@@ -5,5 +5,6 @@
     config.dsn = ENV['SENTRY_DSN']
     config.breadcrumbs_logger = %i[active_support_logger http_logger]
     config.traces_sample_rate = 0
+    config.environment = ENV['SENTRY_ENVIRONMENT'] || "production"
```

Comment:
sentry is using `RAILS_ENV` value, this change would break actual configurations, maybe we can just fall back to `Rails.env`

Reply:
Ah, this was an oversight on my behalf, you're right. I will fix this now.